### ISSUE TITLE
new marks at the top

### ISSUE THREAD

**Issue by primalmotion** [ThePrimeagen/harpoon#148]
Hey! awesone extension! Thanks for this.
I'm wondering if there is a way to make harpoon insert new marks at the top of the quick menu instead of the bottom?
Usually I jump back more often to the marks I just created.

Thanks and sorry if this is possible already, but I couldn't find how.

**Comment by manhoodofhumanity** [comment]
think problem with this is that your marks will be moving every time you add(mark) new file.. so e.g. ui.nav_file(1) leads to new file every time instead of being static. But anyway, think it would be great to people who dont use jumping like that, to add some boolean option to force adding new file to top of buffer)

**Comment by primalmotion** [comment]
yeah, I represent this more as stack of latest marks. that seems more natural to me. An option would be perfect

**Comment by cbochs** [comment]
As a suggestion, you could create a keymap which selects the the last marked file:

```lua
vim.keymap.set("n", "{lhs}", function()
    local marks = require("harpoon").get_mark_config().marks
    require("harpoon.ui").nav_file(#marks)
end)
```

**Comment by manhoodofhumanity** [comment]
With new updates, as i see , harpoon saves its last cursor position, so its comfortable enough now, at least for me
